llvm/utils/TableGen/X86FoldTablesEmitter.cpp

```diff
@@ -508,19 +610,28 @@ void X86FoldTablesEmitter::updateTables(const CodeGenInstruction *RegInst,
           isMemoryOperand(MemOpRec)) {
         switch (I) {
         case 0:
+          assert(!IsBroadcast && "BroadcastTable0 needs to be added");
```

Comment:
What's stopping us adding BroadcastTable0?

Reply:
Nothing. We do not add BroadcastTable0 just b/c it's empty now. If there is an entry should be in BroadcastTable0 one day, the assertion will fail and tell us to add the table.